src/app/blog/blog-card/blog-card.component.css

```diff
@@ -0,0 +1,57 @@
+
```

Comment:
Could you check the styling of this file? There a space before each line of code.